public/apps/configuration/panels/user-list.tsx

```diff
@@ -195,25 +195,25 @@ export function UserList(props: AppDependencies) {
     <>
       <EuiPageHeader>
         <EuiTitle size="l">
-          <h1>Internal users</h1>
+          <h1>Internal accounts</h1>
```

Comment:
I know in the previous thread we have been talked about this, and personally, I don't have any strong opinion for both of the namings. But I was just trying to see if there is any documentation change needed for this. CC: @DarshitChanpura @cwperks @scrawfor99 .

Reply:
Good point. There are bout 25  references  to "Internal Users" in the documentation.  That would have to be updated. Might be easier to just leave as internal accounts